README.rst

```diff
@@ -44,7 +44,8 @@ This documentation contains how-tos, code examples, and an API reference.
 
 Help is also available on the `Twisted mailing list <https://twistedmatrix.com/cgi-bin/mailman/listinfo/twisted-python>`_.
 
-There is also a pair of very lively IRC channels, ``#twisted`` (for general Twisted questions) and ``#twisted.web`` (for Twisted Web), on ``chat.freenode.net``.
+There is also a pair of very lively IRC channels, ``#twisted`` (for general Twisted questions) and ``#twisted.web`` (for Twisted Web),
```

Comment:
according to https://twistedmatrix.com/pipermail/twisted-python/2021-July/065630.html there doesn't seem to be a `#twisted.web` anymore?

Reply:
I have joined the #twisted.web channel just now ...and I am not alone on the channel.
I think that `#twisted.web` is alive.

I think that Tom  only gave a few examples.

But I can see that `#twisted-infra` irc was empty... and on the wiki I have updated it with twisted-dev